src/AzureFunctionPackage/Functions/Common/config.xml

```diff
@@ -3,20 +3,20 @@
     <!-- Roslyn branches (non-vs-deps) (flowing between releases) -->
     <merge from="dev15.0.x" to="dev15.9.x" />
     <merge from="dev15.9.x" to="dev16.0" />
-    <merge from="dev16.0" to="dev16.1-preview2" />
-    <merge from="dev16.1-preview2" to="master" />
```

Comment:
This should ideally be going dev16.0 into dev16.1-preview2 into dev16.1-preview3: the idea is that if we did a servicing fix into Preview 2, we don't want to ship Preview 3 without it.

Reply:
I think I've fixed this. 16.0 -> 16.1P2 -> 16.1P3 -> 16.2